resources/netregexes.js

```diff
@@ -165,6 +165,7 @@ var NetRegexes = {
       8: { field: 'world' },
       9: { field: 'npcNameId' },
       10: { field: 'npcBaseId' },
+      11: { field: 'currentHp' },
```

Comment:
Yeah, I'm not super happy with hp and currentHp, but not sure what to do.  I will follow up after this is merged to make this maxHp and currentHp.